src/converter/bib/BibConversion.js

```diff
@@ -7,31 +7,122 @@ export function convertCSLJSON(source) {
   let bibType = source.type
   let result
 
-  if (bibType === 'article-journal') {
-    result = _convertJournalArticleFromCSLJSON(source)
+  // CSL types: http://docs.citationstyles.org/en/stable/specification.html#appendix-iii-types
+  let typeMapping = {
+    "article": "journal-article",
+    "article-magazine": "journal-article",
+    "article-newspaper": "journal-article",
+    "article-journal": "journal-article",
+    //"bill"
+    "book": "book",
+    //"broadcast"
+    "chapter": "book",
+    "dataset": "data-publication",
+    //"entry"
+    "entry-dictionary": "book",
+    "entry-encyclopedia": "book",
+    //"figure"
+    //"graphic"
+    //"interview"
+    //"legislation"
+    //"legal_case"
+    //"manuscript"
+    //"map"
+    //"motion_picture"
+    //"musical_score"
+    //"pamphlet"
+    "paper-conference": "conference-proceeding",
+    "patent": "patent",
+    //"post"
+    //"post-weblog"
+    //"personal_communication"
+    "report": "report",
+    //"review"
+    //"review-book"
+    //"song"
+    //"speech"
+    "thesis": "thesis",
+    //"treaty"
+    "webpage": "webpage"
+    //NA : "periodical"
```

Comment:
What is the difference between `periodical` and `journal-article`? What would be an example?

Reply:
Like a magazine that occurs regularly. But also not sure how it is actually used in practice. I will start an issue to trigger our partners, which should provide real world examples for each pub type.